app/assets/stylesheets/application.scss

```diff
@@ -20,8 +20,8 @@
  @import 'resources/*';
 
  // import other stylesheets after variables
-@import 'AdminLTE';
 @import 'bootstrap';
+@import 'AdminLTE';
```

Comment:
Since we are removing the second AdminLTE import (from `custom.scss`). We have to move this import after bootstrap. Because AdminLTE depends on and includes bootstrap 4.6 inside it's css, and we want bootstrap 4.6 to have priority over the other bootstrap (version 5.2).